.github/workflows/publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Pyth SDK to crates.io
+
+on:
+  release:
+    types:
+      - created
```

Comment:
Shall we enforce that we can only create releases from `main`? In case someone accidentally creates one on another branch.

Reply:
Unfortunately no, in github actions created by a release I don't think I can get branch name in any way 🤔  But I think it's fine, maybe we want to create a beta in another branch.